scitt-components.md

```diff
@@ -0,0 +1,26 @@
+# SCITT Components
+
+SCITT provides a set of components enabling Supply Chain, Integrity, Transparency and Trust. 
+(See [What Is SCITT][WHAT_IS_SCITT])
+But what are the scenarios that SCITT enables?
+This collection of docs will describe the core components of SCITT, and how it can be enabled and extended to support a breadth of use cases.
```

Comment:
should we address registration policy as well since it's a core component of SCITT?

Reply:
Not sure what you mean.

Reply:
@kkarunakaran89, are you referring to the ingress policy: https://github.com/ietf-scitt/scitt-web/, where a SCITT admin can limit the identity providers?

Reply:
Hi Steve, yup that's one component of it. The reason why I made the comment was due to a lack of reference to registration polices when it comes to SCITT interaction (see interacting with SCITT - https://ietf-scitt.github.io/scitt-web/ ). Wondering if we should capture intent around policies; not just the ingress portion of it but also, for example if an auditor during an audit finds out that the claim was not compliant to the registration polices of the transparency service, then the TS itself can be held liable.